src/scss/components/accordion/_stepper.scss

```diff
@@ -99,4 +100,8 @@ hx-accordion.beta-hxStepper {
       margin-left: 0.5rem;
     }
   }
+
```

Comment:
Please change all `.beta-hxStepXXX` classes above to `.hxStepXXX`.

Reply:
Changes have been done, please review.